blueprints/app/files/.eslintrc.js

```diff
@@ -2,12 +2,39 @@ module.exports = {
   root: true,
   parserOptions: {
     ecmaVersion: 2017,
-    sourceType: 'module'
+    sourceType: 'module',
   },
   extends: 'eslint:recommended',
   env: {
-    browser: true
+    browser: true,
   },
   rules: {
-  }
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+        './ember-cli-build.js',
+        './testem.js',
+        './index.js',
+        'config/**/*.js',
+      ],
+      parserOptions: {
+        sourceType: 'script',
+        ecmaVersion: 2015,
+      },
+      env: {
+        browser: false,
+        node: true,
+      }
+    },
+
+    // test files
+    {
+      files: ['tests/**/*.js'],
```

Comment:
can we exclude `tests/dummy/`?

it's okay if we can't since we haven't done it before either, but would be nice though

Reply:
Yep, added and confirmed.  Works nicely!